expertise/models/bm25/bm25.py

```diff
@@ -85,12 +92,14 @@ def all_scores_helper(self, submissions_dataset):
         csv_scores = []
         for note_id, submission in tqdm(submissions_dataset.items(), total=len(submissions_dataset)):
             reviewer_scores = self.score(submission)
+            if reviewer_scores is None:
```

Comment:
```suggestion
            if not reviewer_scores:
```
Slightly more pythonic.